engine/api/workflow_queue.go

```diff
@@ -143,7 +143,7 @@ func takeJob(ctx context.Context, dbFunc func() *gorp.DbMap, store cache.Store,
 	}
 
 	// Take node job run
-	job, report, err := workflow.TakeNodeJobRun(ctx, tx, store, *p, id, workerModel, wk.Name, wk.ID, infos, hatcheryName)
+	job, report, err := workflow.TakeNodeJobRun(ctx, tx, store, *proj, id, workerModel, wk.Name, wk.ID, infos, hatcheryName)
```

Comment:
need to take the project ?

Reply:
Yes because it call UpdateNodeRunStatus that need it.